### PR TITLE
fix bug: DateTime Implement year() month() day() function.

### PR DESCRIPTION
**Pull request by Aii-Wei**

### CODE REVIEW

File: src/library/datetime.cpp [hunk 1]
```diff
@@ -205,6 +205,15 @@ string Time::asString(const string& format) const {
 DateTime::DateTime(time_t t) {
     value =  (t==0) ? time(NULL) : t;
 }
+int DateTime::day() const {
+    return TimeStruct(value).day();
```
Comment (gittiver) [inline src/library/datetime.cpp:209]: could use timestruct().day()

File: src/library/datetime.cpp [hunk 2]
```diff
@@ -205,6 +205,15 @@ string Time::asString(const string& format) const {
...
+}
+int DateTime::month() const {
+    return TimeStruct(value).month();
```
Comment (gittiver) [inline src/library/datetime.cpp:212]: could use timestruct()

File: src/library/datetime.cpp [hunk 3]
```diff
@@ -205,6 +205,15 @@ string Time::asString(const string& format) const {
...
+}
+int DateTime::year() const {
+    return TimeStruct(value).year();
+}
 int DateTime::hour() const {
```
Comment (gittiver) [inline src/library/datetime.cpp:217]: could use timestruct()

File: src/library/datetime.cpp (hunk 2 above)
Comment (Aii-Wei) [reply src/library/datetime.cpp]: yes, use timestruct(). it is working.

**Pull request merged**